qiskit/algorithms/utils/__init__.py

```diff
@@ -0,0 +1,21 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Common Qiskit algorithms utility functions."""
+
+from .validate_initial_point import _validate_initial_point
+from .validate_bounds import _validate_bounds
```

Comment:
Since we are using these across modules the notion of making them private seems odd. Especially since we expect other algos to use these like VQD now and maybe others going forwards